test/manifestShortcuts.test.js

```diff
@@ -0,0 +1,28 @@
+import favicons from "..";
+import * as fs from "fs";
+import { logo_png, logo_svg } from "./util";
+
+test("manifest should support shortcuts", async () => {
+  expect.assertions(1);
+
+  const result = await favicons(logo_png, {
```

Comment:
May you also add 

```
    icons: {
      android: true,
      appleIcon: false,
      appleStartup: false,
      favicons: false,
      windows: false,
      yandex: false,
    },
```
to test, so we do now blow snapshots.

Reply:
make sense